source/slang/hlsl.meta.slang

```diff
@@ -6936,6 +6936,26 @@ uint countbits(uint value)
     }
 }
 
+__generic <let N : int>
+[__readNone]
+[require(cpp_cuda_glsl_hlsl_metal_spirv, shader5_sm_5_0)]
+vector<uint, N> countbits(vector<uint, N> value)
+{
+    __target_switch
+    {
+    case hlsl:
+        __intrinsic_asm "countbits";
+    case glsl:
+        __intrinsic_asm "bitCount";
+    case metal:
+        __intrinsic_asm "popcount";
+    case spirv:
+        return spirv_asm {OpBitCount $$vector<uint, N> result $value};
+    default:
+        VECTOR_MAP_UNARY(uint, N, countbits, value);
+    }
+}
+
```

Comment:
In glsl.meta.slang, there is a GLSL counter part for this function,
```
public vector<uint,N> bitCount(vector<uint,N> value)
```
Can you make a change so that GLSL version of `bitCount(vector<uint,N>)` simply calls this new function in hlsl.meta.slang?
This will make the GLSL side code slightly simpler and cleaner.